src/modules/flippers/__tests__/Flipper.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { render, screen } from '../../../tests/test-utils';
+import { Flipper } from '../Flipper';
+import { FlippersEnum } from '../constants';
+
+const flipper = FlippersEnum.HideMedRefills;
+
+const flipperActive = {
+  flippers: { flippers: { [flipper]: { key: flipper, value: false } } },
+};
+
+const flipperDisabled = {
+  flippers: { flippers: { [flipper]: { key: flipper, value: true } } },
+};
+
+describe('Testing Flipper component', () => {
+  test('Flipper should show element', () => {
+    render(
+      <Flipper flipper={flipper}>
+        <div>show_test_flipper</div>
+      </Flipper>,
+      { preloadedState: flipperActive },
+    );
+
+    expect(screen.getByText(/show_test_flipper/i)).toBeInTheDocument();
```

Comment:
Please use data-testid if you can )

Reply:
no based on Guiding Principles it is better to use getByText

https://testing-library.com/docs/queries/about#priority

Reply:
no based on Guiding Principles it is better to use getByText

https://testing-library.com/docs/queries/about#priority